src/saml.ts

```diff
@@ -202,6 +202,13 @@ class SAML {
     }
   }
 
+  private getValidationSetting(setting?: string | boolean | undefined): string {
```

Comment:
Shouldn't you be calling this function also at line 1111 of version `269ea7c8` of node-saml's saml.ts
https://github.com/node-saml/node-saml/blob/269ea7c8f1abe9ede03d4c9228d5a150013e9e46/src/saml.ts#L1111-L1139
because value `never` is `truthy`.

As a random side note (not related to this pull request): Aforementioned implementation is surrounded by quite odd "goto label" implementation:
https://github.com/node-saml/node-saml/blob/269ea7c8f1abe9ede03d4c9228d5a150013e9e46/src/saml.ts#L1043-L1140 
but maybe referred code block contains some java-/typecsript magic which requires such implementation instead of just removing `getInresponseTo` label and `break getInResponseTo;` statements (disclaimer: I do not speak java-/typescript/node.js). @markstos

Reply:
Noted - thanks